extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -113,6 +88,40 @@ class AmpLightbox extends AMP.BaseElement {
     this.active_ = true;
   }
 
+  build() {
+    if (this.isBuilt_) {
```

Comment:
I would use `this.container_` to know if it is built or not instead of a new isBuilt variable.